### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by williambout** [Robbatron/drag-relative-path#8]
[Enter steps to reproduce:]

Drag a PNG file in Atom

**Atom**: 1.19.0-beta1 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [drag-relative-path](https://github.com/Robbatron/drag-relative-path) package 1.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of undefined

```
At /Users/williambout/.atom/packages/drag-relative-path/lib/drag-relative-path.coffee:38

TypeError: Cannot read property 'length' of undefined
    at intOrExtDrag (/packages/drag-relative-path/lib/drag-relative-path.coffee:38:32)
    at /packages/drag-relative-path/lib/drag-relative-path.coffee:75:23)
```

### Commands

```
     -9:13.3.0 core:confirm (input.hidden-input)
     -9:12.8.0 core:backspace (input.hidden-input)
     -9:12.3.0 core:confirm (input.hidden-input)
     -9:05.2.0 core:copy (input.hidden-input)
     -9:04.2.0 core:paste (input.hidden-input)
     -9:02.4.0 core:save (input.hidden-input)
  2x -8:26.8.0 core:backspace (input.hidden-input)
     -8:26.2.0 core:save (input.hidden-input)
     -7:55.1.0 find-and-replace:show (input.hidden-input)
     -7:54.2.0 core:confirm (input.hidden-input)
     -7:53.7.0 core:select-all (input.hidden-input)
     -7:51.9.0 core:confirm (input.hidden-input)
     -7:48.2.0 core:save (input.hidden-input)
     -0:33.9.0 settings-view:open (input.hidden-input)
  2x -0:33.2.0 core:paste (div.panels-item)
     -0:28.8.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
atom-ternjs 0.18.3 
autocomplete-paths 1.0.5 
busy-signal 1.4.3 
chester-atom-syntax 0.2.0 
drag-relative-path 1.1.2 
duotone-dark-sea-syntax 2.1.0 
duotone-dark-space-syntax 2.1.0 
emmet 2.4.3 
emmet-atom-plugin 0.0.1 
intentions 1.1.2 
language-babel 2.66.0 
language-ejs 0.4.0 
language-stylus 0.2.0 
linter 2.2.0 
linter-eslint 8.2.1 
linter-ui-default 1.6.1 
minimap 4.28.2 
minimap-git-diff 4.3.1 
minimap-linter 2.0.0 
nucleon 1.0.4 
pigments 0.39.1 
prettier-atom 0.33.0 
project-manager 3.3.5 
react-snippets 0.8.0 
sort-lines 0.14.0 
sync-settings 0.8.2 
wakatime 7.0.3 
```

**Comment by Robbatron** [comment]
@williambout Made some fixes. Can you check if your problem is solved by v1.1.3?

**Comment by williambout** [comment]
@Robbatron I can confirm 1.1.3 solved this issue. Thanks!
